src/configure.ts

```diff
@@ -12,3 +12,5 @@ if (typeof window !== 'undefined') {
   window.Polaris = window.Polaris || {};
   window.Polaris.VERSION = '{{POLARIS_VERSION}}';
 }
```

Comment:
This is technically a change to our public interface, so maybe there's a case for a deprecation warning? @tmlayton what do you think?

Reply:
I don’t think this is a deprecation, rather a way to access the version number from within the library. It’s fine if folks import or reference in a consuming application from window, but maybe I’m missing some part of the thinking @amrocha?

Reply:
Yeah, this isn't a deprecation, it adds a way to access the version number internally.

I don’t think the window global should be deprecated; it should be safe for use by consuming applications. The only case where it _could_ be unavailable is when Polaris is compiled using aggressive tree shaking (ie, sewing-kit). I think that’s an issue for another PR though.